woocommerce/woocommerce
# [CYS] Add two more loading frames to loading sequence

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes https://github.com/woocommerce/woocommerce/issues/41144.

This PR adds two more loaders to the sequence  Mrk6SERPZ4KrFHSjM0a8TK-fi-2748_260986

Getting all frames evenly distributed is tricky because API/iframe in JN and WooExpress loading times will also be a little different, so I decided to keep the current implementation.

https://github.com/woocommerce/woocommerce/assets/4344253/40689a3a-e3bc-4832-bdcb-b95de4afecbb




### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

**Make sure you are testing this with a Jetpack connected instance of WooCommerce, as the AI text completion endpoint can only be called with a Jetpack connected account.**

1. Create a new WooCommerce installation with this version.
2. Make sure to enable `customize-store` feature flag
3. Go to `WooCommerce -> Home` and click `Customize Store` task
4. Click `Design with AI` button
5. Follow through the flow all the way till the loading screen shows up
6.  Confirm the loaders are shown properly




### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
